graph/protocol.go

```diff
@@ -156,6 +167,9 @@ func AddServiceGraphTraffic(toEdge, fromEdge *Edge) {
 		if val, ok := fromEdge.Metadata["grpcErr"]; ok {
 			addToMetadataValue(toEdge.Metadata, "grpcErr", val.(float64))
 		}
+		if responses, ok := fromEdge.Metadata["grpcResponses"]; ok {
```

Comment:
all of these hardcoded strings all over, really need to be constants ("grpcResponses", "httpResponses", "grpcErr", etc, etc). There is going to come a day when someone is going to put a typo in here in these strings and it is going to take us week to find that stupid bug.

Reply:
Blah, I'll think about it.

Reply:
I did this, you're welcome.